js/protoblocks.js

```diff
@@ -222,6 +263,10 @@ class ProtoBlock {
     }
 
     // E.g., forward, right
```

Comment:
you should delete this line.

Reply:
@walterbender Shifted all the respective comments mentioned by you. Thank you for showing me the ropes.